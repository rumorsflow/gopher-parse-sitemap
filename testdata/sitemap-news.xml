<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9">
  <url>
    <loc>http://HOST/</loc>
  </url>
  <url>
    <loc>http://HOST/tools/</loc>
    <lastmod>2015-05-07T19:13:09+09:00</lastmod>
    <news:news>
      <news:publication>
        <news:name>HOST news</news:name>
        <news:language>en</news:language>
      </news:publication>
      <news:publication_date>2015-05-07</news:publication_date>
      <news:title>tools</news:title>
    </news:news>
  </url>
  <url>
    <loc>http://HOST/contribution-to-oss/</loc>
    <lastmod>2015-05-07</lastmod>
    <changefreq>monthly</changefreq>
    <news:news>
      <news:publication>
        <news:name>HOST news</news:name>
        <news:language>en</news:language>
      </news:publication>
      <news:publication_date>2015-05-07T19:20+01:00</news:publication_date>
      <news:title>contribution-to-oss</news:title>
    </news:news>
  </url>
  <url>
    <loc>http://HOST/page-1/</loc>
    <lastmod>2015-05-07T19:13:09+09:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.9</priority>
    <news:news>
      <news:publication>
        <news:name>HOST news</news:name>
        <news:language>en</news:language>
      </news:publication>
      <news:publication_date>2015-05-07T19:20:30.45+01:00</news:publication_date>
      <news:title>page-1</news:title>
    </news:news>
  </url>
</urlset>