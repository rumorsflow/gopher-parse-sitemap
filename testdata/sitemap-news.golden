http://HOST/
always
0.5
http://HOST/tools/
2015-05-07T19:13:09+09:00
always
0.5
HOST news
en
2015-05-07T00:00:00Z
tools
http://HOST/contribution-to-oss/
2015-05-07T00:00:00Z
monthly
0.5
HOST news
en
2015-05-07T19:20:00+01:00
contribution-to-oss
http://HOST/page-1/
2015-05-07T19:13:09+09:00
monthly
0.9
HOST news
en
2015-05-07T19:20:30+01:00
page-1
